api/src/main/java/org/openmrs/module/kenyaemr/reporting/data/converter/definition/evaluator/specialClinics/SpecialClinicsSeroStatusDataEvaluator.java

```diff
@@ -42,7 +42,7 @@ public EvaluatedEncounterData evaluate(EncounterDataDefinition definition, Evalu
         String qry = "select v.encounter_id,\n" +
                 "(case v.sero_status when 703 then 'Positive' when 664 then 'Negative' when 1067 then 'Unknown' else '' end) as sero_status\n" +
                 "from kenyaemr_etl.etl_special_clinics v\n" +
-                "where date(v.visit_date) between date(:startDate) and date(:endDate) and special_clinic = :specialClinic;";
+                "where date(v.visit_date) between date(:startDate) and date(:endDate) and special_clinic_form_uuid = :specialClinic;";
```

Comment:
Special clinic form uuid is a String. Ensure the parameter value is in quotes, preferrably single quotes.